frontend/src/main/components/Courses/CourseForm.js

```diff
@@ -1,7 +1,9 @@
-import React from "react";
+import { React, useState } from "react";
 import { Button, Form } from "react-bootstrap";
 import { useForm } from "react-hook-form";
 import { useNavigate } from "react-router-dom";
+import PersonalScheduleDropdown from "../PersonalSchedules/PersonalScheduleDropdown";
+import { useBackend } from "main/utils/useBackend";
 
 function CourseForm({ initialCourse, submitAction, buttonLabel = "Create" }) {
   // Stryker disable all
```

Comment:
The `// Stryker disable all` here should be paired with `// Stryker restore all` otherwise mutation testing is turned off for this entire file.